frontend/src/pages/Player/RightPlayerPanel/components/NetworkResourcePanel/NetworkResourceLogs.tsx

```diff
@@ -146,6 +146,7 @@ export const NetworkResourceLogs: React.FC<{
 								query={query}
 								selectedCursor={undefined}
 								fetchMoreWhenScrolled={fetchMoreWhenScrolled}
+								bodyHeight="calc(100% - 28px)"
```

Comment:
Where does `28px` come from? Maybe worth adding a comment or assigning to a variable with a meaningful name?

Same goes for the change in `TraceLogs.tsx`.